speaking-portal-project/src/test/test.ts

```diff
@@ -1,20 +1,20 @@
 import {test,expect} from '@jest/globals'
-import { checkFileExistence,checkFileReadAccess,checkWavFile,checkTextFile } from "./file_check";
+import { doesFileExist,isFileReadable,isWavFile,isTextFileValid } from "./file_check";
 
 
 
-test('File Not Found', () => { 
-    expect(() =>checkFileExistence('test.wav')).toThrow('FileNotFound')
+test('File Not Found', () => {
+    expect(() =>doesFileExist('test.wav')).toThrow('FileNotFound')
```

Comment:
sweet thanks adjusting the unit test functions! 

I will work on getting these to run once the PR is merged. The files in the inputs rn are fake files. I will adjust to make sure we are testing the correct testing file for each
edge case.

Reply:
Sounds good, thanks! Let me know if you'd like any help with that, or if you find another test I should include.